examples/rheo/oxidation/in.rheo.oxidation

```diff
@@ -80,7 +80,7 @@ fix             5 rbar rheo/oxidation 1.5 2 1.0
 fix             6 all wall/harmonic ylo EDGE 2.0 1.0 1.0
 fix             7 all gravity 5e-5 vector 0 -1 0
 fix             8 floor setforce 0.0 0.0 0.0
-fix             9 surf add/heat linear 1.1 0.05
```

Comment:
Shouldn't this kind of change require updating the log(s)?

Reply:
You're right. At first I figured it didn't, since the trajectories are the same, but I didn't think of the fact that commands are echoed. Fixed.